haskell/servant/servant-proj.cabal

```diff
@@ -0,0 +1,24 @@
+name:                servant-proj
+version:             0.1.0.0
+-- synopsis:
+-- description:
+category:            Web
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Lib
+  build-depends:       base >= 4.7 && < 5
+                     , servant-server
```

Comment:
I think it could be better to use `servant-server>= 0.18 && < 0.17` to stick to `0.17` @whiplash

Reply:
honestly not sure how stack/cabal handle (semantic?) versioning.

Reply:
I'm sure @jkarni has the answer, but seems that `>= 0.17 && < 0.18` ensure we stick to **0.17**

Refering to https://www.haskell.org/cabal/release/cabal-1.14.0/doc/users-guide/developing-packages.html#package-properties